sites.json

```diff
@@ -728,7 +728,7 @@
    "added":1531345286
    }
```

Comment:
missing comma here

Reply:
updated thanks, 
also their is no problem with site, cheers!